doc/classes/Image.xml

```diff
@@ -762,5 +766,17 @@
 		<constant name="COMPRESS_SOURCE_NORMAL" value="2" enum="CompressSource">
 			Source texture (before compression) is a normal texture (e.g. it can be compressed into two channels).
 		</constant>
+		<constant name="BUMP_TO_NORMAL_ALGORITHM_PARTIAL_DERIVATIVE" value="0" enum="BumpMapToNormalMapAlgorithm">
+			Calculate normals in [code]bump_map_to_normal_map[/code] using the [url=https://en.wikipedia.org/wiki/Partial_derivative]partial derivatives[/url] of the heights. Faster, but potentially noisy.
+		</constant>
+		<constant name="BUMP_TO_NORMAL_ALGORITHM_SOBEL" value="1" enum="BumpMapToNormalMapAlgorithm">
+			Calculate normals in [code]bump_map_to_normal_map[/code] using a [url=https://en.wikipedia.org/wiki/Sobel_operator]Sobel operator[/url]. Slower, but smoother since it looks at more adjacent pixels than [code]BUMP_TO_NORMAL_ALGORITHM_PARTIAL_DERIVATIVE[/code].
```

Comment:
```suggestion
			Calculate normals in [method bump_map_to_normal_map] using a [url=https://en.wikipedia.org/wiki/Sobel_operator]Sobel operator[/url]. Slower, but smoother since it looks at more adjacent pixels than [constant BUMP_TO_NORMAL_ALGORITHM_PARTIAL_DERIVATIVE].
```